test/component/component_test.go

```diff
@@ -69,7 +69,7 @@ func TestComponent(t *testing.T) {
 
 	redisC, err := dp.CreateContainer(ctx,
 		tc.ContainerRequest{
-			Name:       "redis",
+			Name:       "redis-comp-test",
```

Comment:
Changed to something unique because other ways there's a container name conflict with the Harbor Docker Compose setup.

Reply:
Usually we solve one problem or do one enhancement at a time. If name of the container is an issue we should have a dedicated PR to proceed.